lua/cfc_loadouts/client/cl_cfc_loadout.lua

```diff
@@ -1,22 +1,20 @@
 --local UICOLOR = Color( 36, 41, 67, 255 )
-
-local weaponCategorised = {}
-local allWeapons = {}
-
-file.CreateDir("cfc_loadout")
-
-hook.Add( "InitPostEntity", "CFC_Loadouts_ReadyClientCheck", function()
-    net.Start( "CFC_Loadout_InitialSpawn" )
-    net.SendToServer()
-end )
-
-net.Receive( "CFC_Loadout_SendRestrictions", function()
+local weaponCategorised = { }
+local allWeapons = { }
+file.CreateDir( "cfc_loadout" )
+--WIPd
+--hook.Add( "InitPostEntity", "CFC_Loadouts_ReadyClientCheck", function( )
+net.Start( "CFC_Loadout_InitialSpawn" )
+net.SendToServer( )
+
+--end )
+net.Receive( "CFC_Loadout_SendRestrictions", function( )
     allWeapons = list.Get( "Weapon" )
-    local group = LocalPlayer():GetUserGroup()
-    local weaponTable = net.ReadTable()
+    local group = LocalPlayer( ):GetUserGroup( )
+    local weaponTable = net.ReadTable( )
 
     for _, weapon in pairs( allWeapons ) do
```

Comment:
This is a numerically indexed table, so `pairs` isn't the right choice. `ipairs` is the easiest change.

Full-blown performance mode would be:
```lua
-- at the top of the file
rawget = rawget

-- then here
local allWeaponsCount = #allWeapons
for i = 1, allWeaponsCount do
    local wep = rawget( allWeapons, i )
```

Reply:
Not sure why but ipairs breaks it in this case.